yogstation/code/modules/mentor/mentor_verbs.dm

```diff
@@ -70,6 +70,17 @@ GLOBAL_PROTECT(mentor_verbs)
 		msg += "<span class='info'>Mentorhelps are also seen by admins. If no mentors are available in game adminhelp instead and an admin will see it and respond.</span>"
 	to_chat(src, msg, confidential=TRUE)
 
+/client/verb/mrat()
+	set name = "Request Mentor Rat"
+	set category = "Mentor"
+
+	var/mob/living/carbon/human/C = src.mob
```

Comment:
Probably want to verify that they’re a carbon

Reply:
will do

Reply:
done